### DISCUSSION TITLE
Failed tangle, incorrect HTML theme from default Babel Header Args

### DISCUSSION THREAD

**Discussion by okwalerie** [johanwk/elot#61] [General]
Feel free to close. This is for documentation.

## Quick Tip
If you have trouble tangling or exporting to HTML, try clearing your default babel header args.

### Example Issues

When I had the emacs-lisp babel header args set by default to`:results verbatim`:

- `#+call: theme-readtheorg()` failed to add correct `#+SETUPFILE` syntax to the org buffer before exporting to HTML, causing it to be incorrectly styled.
- Many sections of the tangled `.omn` file were surrounded by parentheses and quotation marks like so: `(("%s"))`, and could not be opened by robot.

**Comment by johanwk** [comment]
Thanks, this is useful.

Do you think a change to the code is needed?
